siliconcompiler/report/utils.py

```diff
@@ -76,3 +79,28 @@ def _collect_data(chip, flow, steplist):
             metrics_unit[metric] = metric_unit if metric_unit else ''
 
     return nodes, errors, metrics, metrics_unit, metrics_to_show, reports
+
+
+def _get_flowgraph_path(chip, flow, steplist, summary_table=False):
```

Comment:
rename `summary_table` to something like `only_include_successful` since that is its real usecase